traffic-dashboard/src/components/http-status/http-status.tsx

```diff
@@ -11,18 +11,16 @@ export const HttpStatus: React.FC<IHttpMethodProps> = ({ status }) => {
 };
 
 function statusColor(status: number) {
-  switch (status) {
-    case 201:
-    case 200: {
-      return "success";
-    }
-    case 404: {
-      return "warning";
-    }
-    case 500: {
-      return "error";
-    }
-    default:
-      return "default";
-  }
+  /* maps statuses numbers to color string e.g. 201 -> success */
+  const statusesMap = {
+    2: "success",
+    4: "warning",
+    5: "error",
+  } as any;
+  return statusesMap[status.toString().slice(0, 1)] || "default";
 }
+/*tests (TODO: add real ones)*/
```

Comment:
Please remove the comment.
If you have a TODO you could open an issue for it instead.